src/containers/ChannelHeader.js

```diff
@@ -21,6 +21,11 @@ function ChannelHeader () {
     uiStore.openControlPanel()
   }
 
+  function goBack(e){
+	e.stopPropagation()
+	window.location.hash = (window.lastChannel == undefined) ? '#' : '#/channel/'+window.lastChannel
+  }
+
```

Comment:
A function to return to the last visited channel. The channel name is stored in a global variable named **lastChannel**.